Description: add missing =back to fix "POD ERRORS"
    Around line 87:
        You forgot a '=back' before '=head1'
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2014-10-12

--- a/script/msgconvert
+++ b/script/msgconvert
@@ -84,6 +84,8 @@
 
     Print a brief help message.
 
+=back
+
 =head1 DESCRIPTION
 
 This program will convert the messages contained in the Microsoft Outlook
